ci/glci/model.py

```diff
@@ -480,3 +480,46 @@ def feature_by_name(feature_name: str):
         if feature.name == feature_name:
             return feature
     raise ValueError(feature_name)
+
+
+class OpenRC():
```

Comment:
rather use a dataclass
(see e.g. glci.model.py for examples..)
or: https://docs.python.org/3/library/dataclasses.html for full reference

Reply:
Yep, sounds reasonable to me. Hmm I think for local development we should still keep the option in the constructor to read the values from the environment.